tests/10-vcdm2.js

```diff
@@ -30,10 +33,12 @@ describe('Verifiable Credentials Data Model v2.0', function() {
     const endpoints = new TestEndpoints({implementation, tag: vcApiTag});
     function it2(title, fn) {
```

Comment:
I think this function name slipped past me in review...or predates me... Why not just overload the original `it()`?

Reply:
This comes from @clehner as for overload the original there are a few problems: 1. Javascript doesn't have built in function overloading (typescriot does) 2. even if overloading existed it2 has the same parameters as it meaning it would require changing the parameters. There is an existing issue related to this: https://github.com/w3c/vc-data-model-2.0-test-suite/issues/3

Reply:
Yeah. Fixing the name is all I'm after. Thanks!